integration-tests/deployment/ccip/deploy_home_chain.go

```diff
@@ -375,23 +376,13 @@ func BuildAddDONArgs(
 			OfframpAddress:        offRamp.Address().Bytes(),
```

Comment:
Not related to this PR but we `common.LeftPadBytes` of the onramp address on the offramp, should this also be `common.LeftPadBytes`'d?